spyder/plugins/editor/widgets/editor.py

```diff
@@ -1668,6 +1666,19 @@ def save_if_changed(self, cancelable=False, index=None):
                     return False
         return True
 
+    def compute_hash(self, fileinfo):
+        """Compute hash of contents of editor.
+
+        Args:
+            fileinfo: FileInfo object associated to editor whose hash needs
+                to be computed.
+
+        Returns:
+            int: computed hash.
+        """
+        txt = to_text_string(fileinfo.editor.get_text_with_eol())

```

Comment:
There's no need to use `to_text_string` here because `get_text_with_eol` returns text in unicode.